src/blockcholesky.jl

```diff
@@ -1,37 +1,33 @@
 
 
-##########################
-# Cholesky Factorization #
-##########################
+##########################################
+# Cholesky Factorization on BlockMatrices#
+##########################################
 
-"""
-Funtions to do
-2.check square
-3.check positive definite
-"""
 
 cholesky(A::Symmetric{<:Real,<:BlockArray},
-    ::Val{false}=Val(false); check::Bool = true) = cholesky!(cholcopy(A); check = check)
+    ::Val{false}=Val(false); check::Bool = false) = cholesky!(cholcopy(A); check = check)
 
 
 function b_chol!(A::BlockArray{T}, ::Type{UpperTriangular}) where T<:Real

```

Comment:
Convention in Julia is that internal routines start with `_`. So perhaps rename this function `_block_chol!`?